_inc/client/state/period-toggle/reducer.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { SET_PLAN_DURATION } from 'state/action-types';
+
+const planDuration = ( initialState = 'yearly', action ) => {
+	switch ( action.type ) {
+		case SET_PLAN_DURATION:
+			return action.duration;
+
+		default:
+			return initialState;
+	}
+};
+
+export const reducer = combineReducers( {
+	planDuration,
+} );
+
+/**
+ * Determines if the DevCard should be displayed.
+ * @param  {Object}  state Global state tree
+ * @return {Boolean}       whether the devCard can be displayed
+ */
+export function getPlanDuration( state ) {
+	return state.jetpack.planDuration.planDuration;
```

Comment:
If we were to go with my recommendation earlier for using `_inc/client/state/plans/`, this selector would become more readable:

```suggestion
	return state.jetpack.plans.planDuration;
```